File: JavaScript/01.JavaScript-Basic-February-2021/04.conditionalStatements-Exercise/04.metricConverter.js
function metricConverter(input) {
    let distance = Number(input[0]);
    let fromDim = input[1];
    let toDim = input[2];

    let result;

    if (fromDim === "m") {
        if (toDim === "cm") {
            result = distance * 100;
        } else if (toDim === "mm") {
            result = distance * 1000;
        }
    } else if (fromDim === "cm") {
        if (toDim === "m") {
            result = distance / 100;
        } else if (toDim === "mm") {
            result = distance * 10;
        }
    } else if (fromDim === "mm") {
        if (toDim === "m") {
            result = distance / 1000;
        } else if (toDim === "cm") {
            result = distance / 10;
        }
    }
    console.log(result.toFixed(3));
} 

metricConverter(["12", "mm", "m"]);
metricConverter(["150", "m", "cm"]);
metricConverter(["45", "cm", "mm"]);

File: Python/01.Python-Basic-April-2019/05.conditional_statements_exercise/04.metric_converter.py
number = float(input())

enter_dim = input()
exit_dim = input()

if enter_dim == 'mm' and exit_dim == 'm':
    print(f'{number / 1000:0.3f}')
elif enter_dim == 'mm' and exit_dim == 'cm':
    print(f'{number / 10:0.3f}')
elif enter_dim == 'cm' and exit_dim == 'm':
    print(f'{number / 100:0.3f}')
elif enter_dim == 'cm' and exit_dim == 'mm':
    print(f'{number * 10:0.3f}')
elif enter_dim == 'm' and exit_dim == 'cm':
    print(f'{number * 100:0.3f}')
elif enter_dim == 'm' and exit_dim == 'mm':
    print(f'{number * 1000:0.3f}')
